docs/install_occlum_packages.md

```diff
@@ -0,0 +1,102 @@
+# Install Occlum with Popular Package Managers
+
+Occlum can be easily installed with popular package managers like [APT](https://en.wikipedia.org/wiki/APT_(software)) and [RPM](https://en.wikipedia.org/wiki/RPM_Package_Manager). This document walks you through the steps to install Occlum on popular Linux distributions like Ubuntu and CentOS using package managers.
+
+## Prerequisite
+
+**Install enable_RDFSBASE Kernel Module**
+
+If the Kernel version is before `v5.9`, please follow [this README](https://github.com/occlum/enable_rdfsbase/blob/master/README.md) to install `enable_rdfsbase` kernel module.
+
+
+## Install Occlum with RPM on CentOS 8.1
```

Comment:
Update CentOS 8.1 to 8.2 to align with the latest status.

Reply:
Done. Thanks for your review.